### PR TITLE
feat: done landingPage

### PR DESCRIPTION
**Pull request by Shailseen**
### Landing page
Using component library for some components
image

### CODE REVIEW

**Review by ianuragsharma — APPROVED** [review]
Left a comment do look into it.
Rest looks good.

File: src/routes/landingPage/landingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <p>
+            Manage your daily tasks and workflow in a modern way and boost your
+            efficiency without any efforts.
+          </p>
+        </div>
+        <div className="authentication-container flex-col text-align-left">
+          <button className="button-style-none solid-button disable-hover button-style">
+            Join Now
+          </button>
+          <a className="login-link" href="">
```
Comment (ianuragsharma) [inline src/routes/landingPage/landingPage.jsx:24]: use LINK element instead of anchor tag

Comment (Shailseen) [reply src/routes/landingPage/landingPage.jsx:24]: Sure, i will add this after adding React-router.

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,17 @@
+import { Routes } from "react-router-dom";
+import { Route } from "react-router-dom";
+import { BrowserRouter } from "react-router-dom";
```
Comment (sagargse) [inline src/App.js]: Can you write this using comma-separated in a single line?

**Review by sagargse — APPROVED** [review]
Code looks clean to me just do the suggested changes.
And go ahead.

File: src/App.js (hunk 2 above)
Comment (Shailseen) [reply src/App.js]: okay

**Pull request merged**